library/src/amd_detail/rocblaslt/src/rocblaslt_mat.cpp

```diff
@@ -442,6 +446,7 @@ rocblaslt_status
     std::vector<const void*>        scaleD_vec;
     std::vector<const void*>        scaleE_vec;
     std::vector<const void*>        scaleAlpha_vec;
+    std::vector<void*>              amaxD_vec;
```

Comment:
use const void* unless there is any special reason.

Reply:
done

Reply:
Oh, D_vec, E_vec and amaxD_vec are used as output for RocblasltContractionProblem they are all void* instead of const void*